js/footerinclude.js

```diff
@@ -0,0 +1,40 @@
+const footerTemplate = document.createElement('template');
+
+footerTemplate.innerHTML = `
+<div class="footer-main">
+<div class="footer-text" id="info">
+  <h1>Databased</h1>
+  <h4>
+    <a class="footer-link" href="https://csa.iisc.ac.in">
+      The Department of Computer Science and Automation (CSA)
+    </a>
+    <br />
+  </h4>
+  <h5>
+    <a class="footer-link" href="https://iisc.ac.in">
+      Indian Institute of Science, Bengaluru - 560012, INDIA.
+    </a>
+  </h5>
+  <br>
+  <h6>
+    Copyright © <span class="copyright">2023</span> by The Department of Computer Science and Automation, Indian
```

Comment:
We can remove CSA & IISc from the Copyright text as this is a repetition from above.